Flixel Features/FlxSound/source/MenuState.hx

```diff
@@ -0,0 +1,122 @@
+package;
+
+import flixel.addons.ui.FlxUINumericStepper;
+import flixel.FlxG;
+import flixel.addons.ui.FlxUIButton;
+import flixel.addons.ui.FlxUITypedButton;
+import flixel.addons.ui.FlxUICheckBox;
+import flixel.addons.ui.FlxUICursor;
+import flixel.addons.ui.FlxUIState;
+import flixel.addons.ui.FlxUIText;
+import flixel.addons.ui.interfaces.IFlxUIWidget;
+import flixel.util.FlxColor;
+import openfl.Assets;
+
+/**
+ * A FlxState which can be used for the game's menu.
+ */
+class MenuState extends FlxUIState
+{
+	private var loop_music:Bool = false;
+	private var music_volume:Float = 0.5;
+	private var sfx_volume:Float = 0.5;
+	private var loop_count:Int = 0;
+	
+	/**
+	 * Function that is called up when to state is created to set it up. 
+	 */
+	override public function create():Void
+	{
+		_xml_id = "menu";
+		super.create();
+		
+		var butt_music:FlxUIButton = cast _ui.getAsset("butt_music");
```

Comment:
Oh I've always done that, guess I have the mind of a teenager :P I can change to btn_ if you insist.